### ISSUE TITLE
Segmentation fault when a regular MKV file, not generated by RAWcooked, is passed

### ISSUE THREAD

**Issue by retokromer** [MediaArea/RAWcooked#338]
Passing to RAWcooked version 21.01 a regular Matroska/FFV1 file rather than a RAWcooked one
```
rawcooked file_ffv1.mkv
```
I  got
```
Time=00:00:00 (0%)    Segmentation fault: 11
```
In my opinion, the error message should be “something” more precise, when the file is a valid file, but not one generated by RAWcooked. And the program should not exit with a segmentation  fault.

**Comment by JeromeMartinez** [comment]
A bit linked to https://github.com/MediaArea/RAWcooked/issues/118, but I definitely need an error message instead of a crash, then maybe https://github.com/MediaArea/RAWcooked/issues/119.

**Comment by retokromer** [comment]
I forgot about https://github.com/MediaArea/RAWcooked/issues/118#issuecomment-477949546, when I made the “mistake” this afternoon and therefore I filed this issue. A message rather than a crash would be useful. Thank you!

**Comment by retokromer** [comment]
>A message rather than a crash would be useful.

@JeromeMartinez : Could this be fixed in the next release? Thank you!

**Comment by retokromer** [comment]
This bug is fixed in version `21.09`. It gives now the error message:
```
Error: No reversibility data found.
```
Thank you!
